dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -86,7 +87,7 @@ protected static String getEventType(String source, String location) {
             return "";
     }
 
-    protected static ThreadLocal<BurstDateFormat> dateFormatTL = new ThreadLocal<BurstDateFormat>() {
+    public static ThreadLocal<BurstDateFormat> dateFormatTL = new ThreadLocal<BurstDateFormat>() {
```

Comment:
Yes, it's used by all the other jsonify methods (jsonifyAudit, jsonifyFFDC, etc) - also, AccessLogSource now also uses it (hence the change from protected->public) since I moved the datetime field into the list of formatters: https://github.com/OpenLiberty/open-liberty/pull/11189/files#diff-83645dd44864a951f486a50cbd6b6e27R375

(per this review comment https://github.com/OpenLiberty/open-liberty/pull/11189#discussion_r414660933)